week-2/implement/lookup.js

```diff
@@ -1,17 +1,11 @@
-function createLookup() {
-  // implementation here
+function createLookup(givenArray) {
+  let newObject = {};
+  for (let element of givenArray) {
+    newObject[element[0]] = element[1];
+  }
+  return newObject;
 }
-
+console.log(createLookup([["abc", "def"]]));
```

Comment:
Again no need to push console.logs is should be used to test and then just remove them before you push.